### PR TITLE
callstack and scope panes open on pause

### PR DESCRIPTION
**Pull request by liorpr**
Associated Issue: #1280 

Right Sidebar gets the pause data from the state, and the Accordion items now have a shouldOpen prop. Just like @jasonLaster suggested.

### CODE REVIEW

**Comment by jasonLaster** [comment]
> when the pause ends, should they close?

sorry, i missed this earlier. I think, they can stay open :)

File: src/components/Accordion.js [hunk 1]
```diff
@@ -18,6 +18,17 @@ const Accordion = React.createClass({
       created: [] };
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    const opened = [...this.state.opened];
+    const newOpened = opened.map((isOpen, i) => {
+      const shouldOpen = this.props.items[i].shouldOpen;
+
+      return isOpen ? isOpen : shouldOpen && shouldOpen();
```
Comment (jasonLaster) [inline src/components/Accordion.js]: Hmm perhaps isOpen || (shouldOpen && sjouldOpen()

File: src/components/Accordion.js [hunk 2]
```diff
@@ -18,6 +18,17 @@ const Accordion = React.createClass({
       created: [] };
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    const opened = [...this.state.opened];
```
Comment (jasonLaster) [inline src/components/Accordion.js]: What's this doing :)

Comment (liorpr) [inline src/components/Accordion.js]: it's being used in handleHeaderClick down below, seems like copying to a new array, although  unnecessary here cause I use .map (realizing  just now :)  )

**Pull request merged**